README.md

```diff
@@ -24,6 +24,11 @@ Ensure you have Casbin's system dependencies installed by:
 sudo apt install gcc libpcre3 libpcre3-dev
```

Comment:
Isn't this also for Linux?

Reply:
apt isn't available on alpine linux image by default where as apk package manager is only availble but the dependency libraries names are not same in alpine linux package repository.